### PR TITLE
Works section

### PR DESCRIPTION
**Pull request by Wes-Isaac**
Added about me section on the index page and put in a style for it in the corresponding css file.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -39,6 +36,172 @@ <h1>Hey there. I’m Isaac.
            <a href="#"> <i class="fal 2x fa-chevron-circle-down fa-2x dropdown"></i></a>
         </section>
     </main>
-    
+    <section>
+        <header>
+            <h1 class="section-header">My Recent Works</h1>
+            <div class="indicator"></div>
+        </header>
```
Comment (DanteAlonsoHT) [inline index.html:43]: - [ ] [OPTIONAL] Please, apply responsive design techniques, your content for "My Recent Work" should be centered.

image

File: index.html [hunk 2]
```diff
@@ -39,6 +36,172 @@ <h1>Hey there. I’m Isaac.
...
+                <div class="text">
+                    <h3>Multi-Post Stories Gain+Glory</h3>
+                    <ul class="skillsbar">
+                        <li>Ruby on rails</li>
+                        <li>css</li>
+                        <li>JavaScript</li>
+                        <li>html</li>
+                    </ul>
+                    <button>See Project</button>
+                </div>
```
Comment (DanteAlonsoHT) [inline index.html:56]: - [ ] You did a great job, just you can improve it a little bit more -> removing spaces where you can see red color, and adding more space where you can see blue color.

Figma design | Your design

image

File: index.html [hunk 3]
```diff
@@ -39,6 +36,172 @@ <h1>Hey there. I’m Isaac.
...
+        </div>
+    </section>
+
+    <section>
+        <div class="aboutme-text">
+            <div class="head-ele">
+
+                <h1 class="aboutme-header">About me</h1>
+                <div class="bg-img-1"></div>
+            </div>
```
Comment (DanteAlonsoHT) [inline index.html:132]: - [ ] [OPTIONAL] For your image ".bg-img-1" I recommend you to use position absolute inside a position relative to center better your "About me" title avoiding adding something together with your "About me" title. For instance:

```
// index.html
<div class="head">
  <div class="bg-img">
</div>

// style.css
.head {
 position: relative;
}

.bg-img {
  position: absolute;
  right: 0;
  top: 0;
}
```

Also, probably the issue might be solved using `text-align: center`, you can try to find what is the right way for you.

image

**Review by DanteAlonsoHT — CHANGES_REQUESTED** [review]
<h1> Status: Required Changes 🚫 </h1>

Hi @Wes-Isaac,

Amazing job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️


- [x] Improve the description in your Pull Request, explain what you did, what is the main scope, which files do you changed or added, what you set up in your project in this pull request, and so on.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by barackm — APPROVED** [review]
Hi @Wes-Isaac, 👋 

# STATUS: Approved  🎉 
Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉  🎉 🎉 

- [ ] [OPTIONAL] It could be nice if you work on the optional changes by the previous reviewer since it's important what he mentioned 😉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Wes-Isaac** [comment]
Added an about me section.

**Pull request merged**